azkaban-common/src/test/java/azkaban/executor/ExecutionFlowDaoTest.java

```diff
@@ -232,13 +212,11 @@ public void testFetchRecentlyFinishedFlows() throws Exception {
 
   @Test
   public void testFetchEmptyRecentlyFinishedFlows() throws Exception {
-    final ExecutableFlow flow1 = createTestFlow();
-    this.executionFlowDao.uploadExecutableFlow(flow1);
-    flow1.setStatus(Status.SUCCEEDED);
-    flow1.setEndTime(DateTimeUtils.currentTimeMillis());
+    final ExecutableFlow flow1 = createExecution(System.currentTimeMillis(), Status.SUCCEEDED);
+    flow1.setEndTime(System.currentTimeMillis());
     this.executionFlowDao.updateExecutableFlow(flow1);
     //Todo jamiesjc: use java8.java.time api instead of jodatime
-
+    
     //Mock flow finished time to be 2 min ago.
     DateTimeUtils.setCurrentMillisOffset(-FLOW_FINISHED_TIME.toMillis());
```

Comment:
Did we consider the alternative
_long twoMinutesAgo = currentTime - FLOW_FINISHED_TIME.toMillis();
setEndTime(twoMinutesAgo)_ because it would be more readable?

Reply:
Fixed it.